src/sentry/services/hybrid_cloud/organization/impl.py

```diff
@@ -92,6 +92,56 @@ def check_membership_by_email(
 
         return serialize_member(member)
 
+    def check_membership_by_invite_token(
+        self, organization_id: int, org_member_id: int, invite_token: str
```

Comment:
```suggestion
        self, organization_id: int, member_id: int, invite_token: str
```

We use both `org_member_id` and `organization_member_id` which could be confusing. Should we try to be more consistent on the name we use for these parameters?